app/scripts/services/catalog.js

```diff
@@ -21,10 +26,41 @@ angular.module("openshiftConsole")
                                   APIService.apiInfo(serviceInstancesVersion) &&
                                   APIService.apiInfo(servicePlansVersion);
 
+    var cachedCatalogItems;
+
+    var clearCachedCatalogItems = function() {
+      Logger.debug('ProjectsService: clearing catalog items cache');
+      cachedCatalogItems = null;
+    };
+
+    AuthService.onUserChanged(clearCachedCatalogItems);
+    AuthService.onLogout(clearCachedCatalogItems);
+
     var isTemplateServiceBrokerEnabled = function() {
       return !!Constants.TEMPLATE_SERVICE_BROKER_ENABLED;
     };
 
+    var getCatalogItems = function(forceRefresh) {
```

Comment:
Looks good. Do you mind updating landing page to use this?

Reply:
sure thing